TreeTracker/UI/Views/View Trees/ViewTreesViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ViewTreesViewController.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 10/27/20.
+//  Copyright © 2020 Greenstand. All rights reserved.
+//
+
+import UIKit
+
+class ViewTreesViewController: UIViewController {
+    var trees: [Tree] = [] {
+        didSet {
+            viewTreesCollectionview.reloadData()
+        }
+    }
+    var viewModel: ViewTreesViewModel? {
+        didSet {
+            viewModel?.viewDelegate = self
+            title = viewModel?.title
+        }
+    }
+    @IBOutlet weak var viewTreesCollectionview: UICollectionView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        viewTreesCollectionview.delegate = self
+        viewTreesCollectionview.dataSource = self
+//      collectionView.register(ViewTreesCollectionViewCell.self, forCellWithReuseIdentifier: "ViewTreesCollectionViewCell")
+        viewModel?.fetchTrees()
+    }
+}
+// MARK: - CollectionView cell implementation
+extension ViewTreesViewController: UICollectionViewDelegate, UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return trees.count
+    }
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+
```

Comment:
We can remove this function if we don't need it.

Reply:
we may want to use DidSelectItem func later on.

Reply:
I removed